loky/backend/context.py

```diff
@@ -52,13 +52,20 @@ def assert_spawning(obj):
             )
 
     def get_context(method="loky"):
-        return LokyContext()
+        if method == "loky":
+            return LokyContext()
+        elif method == "loky_no_main":
+            if sys.platform == "win32":
+                return LokyContext()
```

Comment:
Why the special case under windows? Ideally we would like to make this feature benefit windows users as well.